docs/src/man/structure.md

```diff
@@ -20,13 +20,14 @@ The `Bio.Structure` module provides functionality to manipulate macromolecular s
 To download a PDB file:
 
 ```julia
+# Stored in the current working directory by default
 downloadpdb("1EN2")
 ```
 
 To parse a PDB file into a Structure-Model-Chain-Residue-Atom framework:
 
 ```julia
-julia> struc = read(filepath_1EN2, PDB)
+julia> struc = read("/path/to/pdb/file.pdb", PDB)
```

Comment:
For reference, this was initially set as `filepath_1EN2` to pass the doctests but I don't think they are being tested here so this can stay as `"/path/to/pdb/file.pdb"`.